revolv/templates/base/base.html

```diff
@@ -49,7 +49,7 @@
     {% endblock %}
 </head>
 
-<body {% block body_class %}{% endblock%}>
+<body onscroll="myfunction()" {% block body_class %}{% endblock%} id="{% if 'project' in request.get_full_path %}project{% else %}bye{% endif %}">
```

Comment:
What is the purpose of `myfunction`? `myfunction` is not a good name, because it doesn't describe what the function does. Please name it something more descriptive.